zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -16,6 +17,7 @@
 REPLICASET_TYPE = 'kube_replicaset'
 STATEFULSET_TYPE = 'kube_statefulset'
 DAEMONSET_TYPE = 'kube_daemonset'
+POSTGRESDB_TYPE = 'postgresql_database'
```

Comment:
It is a little confusing since we are syncing Spilo clusters specifically, so ``postgresql_database`` seems more generic than what we do here. I was thinking may be use ``database`` as type and add extra attributes to identify ``spilo`` and ``postgresql``. What do you think?